src/main/resources/helptexts/alternativeIdentifiers.html

```diff
@@ -1,5 +1,17 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>Identifier</h1>
```

Comment:
Please call this `Alternative Identifier`

Reply:
done